phpBB/install/startup.php

```diff
@@ -81,4 +81,5 @@ function phpbb_include_updated($path, $phpbb_root_path, $optional = false)
 
 $phpbb_installer_container = $phpbb_installer_container_builder
 	->with_config_path($config_path)
+	->with_custom_parameters(array('cache.driver.class' => 'phpbb\cache\driver\file'))
```

Comment:
I don't like this patch because it's a hack.... but it should work and I don't see any other "small" solution :/